docs/docs/installation/rasa-pro/installation.mdx

```diff
@@ -96,36 +83,36 @@ For virtual environments:
 - Linux and macOS: `$VIRTUAL_ENV/pip.conf`
 - Windows: `%VIRTUAL_ENV%\pip.ini`
 
-
 ```
 [global]
 extra-index-url = https://europe-west3-python.pkg.dev/rasa-releases/rasa-plus-py/simple/
-
 ```
 
 Finally, you should be able to run `pip install rasa-plus`.
 
 #### Installing with `poetry`
 
-To install `rasa-plus` with `poetry`, you will need to associate the Artifact Registry URL with `rasa-plus` before installing it.
-Note that you must upgrade poetry to the latest minor (`1.2.0`) in order for `poetry` to work with the GCP authentication set-up.
-Proceed with the following steps:
+:::note
+The following configuration applies to the current stable version of `poetry`,
+which is `1.6.1` at this time.
```

Comment:
can you skip the patch part of the version? it should work with any version on 1.6.x, right?